src/navigation/services/buy-crypto/screens/BuyCryptoOffers.tsx

```diff
@@ -738,11 +729,7 @@ const BuyCryptoOffers: React.FC = () => {
           destinationChain,
         );
 
-        Linking.openURL(paymentUrl)
-          .then(() => {
-            navigation.goBack();
```

Comment:
You forgot that the view should go back once the inapp browser is open

Reply:
Thank you!